### PR TITLE
Add task changes

### PR DESCRIPTION
**Pull request by FIREDAN99**
please do not forget provide demo links in PR description
- [DEMO LINK](https://FIREDAN99.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://FIREDAN99.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+[data-qa="big"] {
```
Comment (VitaliyBondarenko1982) [inline src/style.css:1]: get used to style all elements using classes
and check this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
Screenshot from 2023-04-06 13-23-02

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px ;
+  line-height: 22px;
+  background-size: 19px;
+  padding-left: 62px;
+  font-weight: 400;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:17]:
```suggestion
  padding-left: 62px;
```
resundant - 400 is default font-weight

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  padding-top: 3px;
+}
+
+[data-qa="small"]::placeholder {
+  font-family: Avenir, Book, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```
Comment (VitaliyBondarenko1982) [inline src/style.css:78]: check styles and remove repeatable, and which not necessary to set again ( small input, hover, focus states)
for example make class search, and search-small for small form, where reset only classes, which different from big form

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+[data-qa="big"]::placeholder:focus {
+  margin: 27px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+[data-qa="small"] {
+  font-family: Avenir, Book, sans-serif;
+  box-sizing: border-box;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:55]: move to * selector
```
* {
  box-sizing: border-box;
}

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
+[data-qa="big"] {
+  font-family: Avenir, Book, sans-serif;
+  margin-top: 12px;
+  box-sizing: border-box;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:4]: same. move to * selector

* {
  box-sizing: border-box;
}

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
also remove outline for inputs
```
outline: none
```

Screenshot from 2023-04-06 13-20-40

**Comment by FIREDAN99** [comment]
add task changes2

**Pull request closed without merging**